Mixin/UserInterface/Controllers/Home/Apps/Drag/HomeAppsFolderInteraction.swift

```diff
@@ -0,0 +1,61 @@
+import Foundation
+
+protocol HomeAppsFolderInteraction {
```

Comment:
`dragInteraction` `item` `wrapperView` are never been written in the app. No need to be settable.